src/models/teams.rs

```diff
@@ -59,11 +59,16 @@ impl ProjectPermissions {
         }
 
         if let Some(member) = project_team_member {
+            if member.accepted {
+                return Some(member.permissions);
+            }
             return Some(member.permissions);
```

Comment:
As written, this change has no effect on the behavior of the code - it just always returns `Some(member.permissions)`. Did you maybe intend to only return that when the member has accepted?

Reply:
Whoops- this is an artifact from that test I commented out.